proxy/manager_test.go

```diff
@@ -62,3 +66,57 @@ func TestProxyHandleRequestError(t *testing.T) {
 		})
 	})
 }
+
+func TestIsReleaseCalendarURl(t *testing.T) {
+	Convey("Given a list of URLs", t, func() {
+		releaseCalendarEndpoints := []TestCase{
```

Comment:
Given that all the cases in this group are expected to be true, and all the other cases in the other group are expected to be false, this can be simplified a lot by just having two different Convey tests, you don't even need to create a `TestCase` struct for this.